ComponentTextKitApplicationTests/CKTextKitTests.mm

```diff
@@ -135,4 +137,22 @@ - (void)testStringsWithVariableAttributes
   XCTAssert(checkAttributes(attributes, { 100, 100 }));
 }
 
+- (void)testRectsForRangeBeyondTruncationSizeReturnsNonZeroNumberOfRects
+{
+  NSAttributedString *attributedString =
+  [[NSAttributedString alloc]
+   initWithString:@"90's cray photo booth tote bag bespoke Carles. Plaid wayfarers Odd Future master cleanse tattooed four dollar toast small batch kale chips leggings meh photo booth occupy irony.  " attributes:@{CKTextKitEntityAttributeName : [[CKTextKitEntityAttribute alloc] initWithEntity:@"entity"]}];
+
+  CKTextKitRenderer *renderer =
+  [[CKTextKitRenderer alloc]
+   initWithTextKitAttributes:{
+     .attributedString = attributedString,
+     .maximumNumberOfLines = 1,
+     .truncationAttributedString = [[NSAttributedString alloc] initWithString:@"... Continue Reading"]
+   }
+   constrainedSize:{ 100, 100 }];
+
+  XCTAssert([renderer rectsForTextRange:NSMakeRange(0, attributedString.length) measureOption:CKTextKitRendererMeasureOptionBlock].count > 0);
```

Comment:
Should we add test cases for the updated conditional? Basically if the clamped range location is not found or its length is 0.